cost/budget_report_alerts/budget_report_alerts.pt

```diff
@@ -46,6 +46,12 @@ parameter "param_use_groups" do
   description "Use this parameter to specify how to group the data"
 end
 
+parameter "param_filter" do
+  type "list"
+  label "Filter Group By Dimension(s)"
+  description "Filter by dimension=value like 'Cloud Vendor=AWS' & 'Cloud Vendor Account=Account1,Account2'. Multiple filters with multiple values can be provided. Group Filter by Dimension only support dimensions   provided in the group previously defined in the budget. Use this field to alert on the dimension values specified in the list"
```

Comment:
This description is too long; detailed instructions can be provided in the README. This should be shortened and concise, and follow proper grammar rules.